server/xpub-server/entities/manuscript/resolvers/removeUploadedManuscript.js

```diff
@@ -11,6 +11,7 @@ async function removeUploadedManuscript(_, vars, { user }) {
 
   if (manuscriptUploadIndex > -1) {
     manuscript.files.splice(manuscriptUploadIndex, 1)
+    // TODO: Should we remove the content from S3 ???
```

Comment:
If we do overwrite the exiting manuscript in s3 - then we need to test that it does

Reply:
#1163